### ISSUE TITLE
[bug] 1.19.2 6.0.5 Loaded 0 jetpacks from the server

### ISSUE THREAD

**Issue by CKY-** [BlakeBr0/IronJetpacks#78]
**Describe the bug**
jetpacks are not loading 
[25Sep2023 12:56:09.514] [Render thread/INFO] [Iron Jetpacks/]: Loaded 0 jetpacks from the server

this works on 1.19.2 6.0.4

**To Reproduce**

**Expected behavior**

**Screenshots / Scripts / Logs**
Please add your logs and scripts (if applicable).
here is the log there is not much in it other then the above line 
[https://gist.github.com/CKY-/abb1c6bd4f5216c4a3c07750faad436f](https://gist.github.com/CKY-/abb1c6bd4f5216c4a3c07750faad436f)

**Versions (please complete the following information):**
 - Minecraft: 1.19.2
 - Forge: 43.3.0
 - Cucumber: 1.19.2 - 6.0.8
 - Iron Jetpacks: 1.19.2 - 6.0.5

**Comment by BlakeBr0** [comment]
Either downgrade or find the mod in your pack that is breaking it

**Comment by BlakeBr0** [comment]
You can probably just downgrade. I'm not sure which mod is causing it not to work, but a mod seems to be completely replacing the vanilla recipe manager which is quite annoying.

**Comment by CKY-** [comment]
I disabled HammerLiB and it magically works... 
Now to figure out why.. And figure out how to report this to that author.

**Comment by Zeitheron** [comment]
@BlakeBr0 Hey, I have just analyzed your code and [RecipeManagerMixin](https://github.com/BlakeBr0/Cucumber/blob/1.19/src/main/java/com/blakebr0/cucumber/mixin/RecipeManagerMixin.java) seems to be the only place where you create and post the `RecipeManagerLoadingEvent`.

I also have a dynamic recipe registry in HammerLib which does a very similar thing, see [RecipeManagerMixin](https://github.com/dragon-forge/HammerLib/blob/1.19.2/src/main/java/org/zeith/hammerlib/mixins/RecipeManagerMixin.java).
Your problem is likely related to KubeJS, not HammerLib.

KubeJS replaces Minecraft's recipe manager to add JS support (which is stupid, I know, and could be done without replacing the vanilla system as it does) (thanks, Lat.)
But my suggestion is to do a second mixin for when KubeJS is installed. See my workaround here: [RecipeEventJSMixin](https://github.com/dragon-forge/HammerLib/blob/1.19.2/src/main/java/org/zeith/hammerlib/mixins/bs/kubejs/RecipeEventJSMixin.java).

I hope this helps!

**Comment by BlakeBr0** [comment]
@Zeitheron thanks for the heads up. I'll have to investigate that further.

**Comment by BlakeBr0** [comment]
I think I'm just going to remove all the dynamic recipe stuff at some point. It's been a major pain point in development for a while (which makes sense because it's hacky). This stuff is a holdover from 1.12 when recipes were registered in-code instead of datapacks anyways, and I think pack devs are a little more accustomed to having to use datapacks now.

**Comment by Zeitheron** [comment]
I haven't really seen any other reports of missing recipes after adding compatibility mixin for KubeJS, so I would say that should help with keeping in-code recipes.
No other mod I know replaces vanilla recipe manager, so that should be about it for patching this issue.

I have also started when recipes were added via code, and just can't get used to writing JSON files for every item. (Just too much writing and guessing, and potential problems when refactoring/renaming things)
All of my mods use HammerLib's recipe registration method and that isn't going to change.

But it's ultimately up to you to decide how you want to tackle this issue.
